stan/math/prim/prob/multi_student_t_lpdf.hpp

```diff
@@ -94,9 +95,11 @@ return_type_t<T_y, T_dof, T_loc, T_scale> multi_student_t_lpdf(
     check_finite(function, "Location parameter", mu_vec[i]);
     check_not_nan(function, "Random variable", y_vec[i]);
   }
-  check_symmetric(function, "Scale parameter", Sigma);
+  const auto& Sigma_ref = to_ref(Sigma);
```

Comment:
Can't mu/nu/y be references?

Reply:
Actually yeah nu gets used in a few places. How does this work? y and mu go through vector_seq_view though so they should be fine.

Reply:
Simple - `nu` is always a scalar.